oauth2_provider/migrations/0006_application_enable_pkce.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I've tested this migration locally. Even though we've overridden the Application model and added other fields to it in pie, it seems to work fine. Django runs the migrations for pie's oauth app as well as the oauth2_provider app, but it must be faking the second one.